hack/run-against-snapshot.sh

```diff
@@ -13,24 +14,21 @@ local_git_version=$(git describe --tags --always --dirty)
 IMAGE_VERSION=${IMAGE_VERSION:-$local_git_version}
 
 snap_tmp_dir=$(mktemp -d -t ghw-snap-test-XXX)
+# needed to enabled PRESERVE and EXCLUSIVE (see README.md)
+mkdir -p "$snap_tmp_dir/root"
 
-echo "extracting snapshot $SNAPSHOT_FILEPATH to $snap_tmp_dir ..."
-tar -xf $SNAPSHOT_FILEPATH -C $snap_tmp_dir
+echo "copying snapshot $SNAPSHOT_FILEPATH to $snap_tmp_dir ..."
+cp -L $SNAPSHOT_FILEPATH $snap_tmp_dir
 
 echo "building Docker image with ghwc ..."
 
-docker build -f $root_dir/Dockerfile -t $ghwc_image_name:$IMAGE_VERSION $root_dir
-
-echo "gathering symlink dirs in extracted snapshot to host volume mount ..."
-
-linkdirs=""
-for dirlink in `find -L $snap_tmp_dir -xtype l`; do
-    sourcelink=$( readlink $dirlink )
-    sourcelink=$( echo $sourcelink | sed "s/\/tmp\/ghw-snapshot[[:digit:]]\+/$tmp_snap_dir/")
-    target=$( echo "$dirlink" | sed "s/\/tmp\/ghw-snap-test-.../\/host/")
-    linkdirs+=" --mount type=bind,source=$sourcelink,destination=$target"
-done
+${CONTAINER_RUNTIME} build -f $root_dir/Dockerfile -t $ghwc_image_name:$IMAGE_VERSION $root_dir
 
 echo "running ghwc Docker image with volume mount to snapshot dir ..."
 
-docker run -it -v $snap_tmp_dir:/host $linkdirs -e GHW_CHROOT="/host" $ghwc_image_name:$IMAGE_VERSION 
+${CONTAINER_RUNTIME} run -it -v $snap_tmp_dir:/host:z \
```

Comment:
I see that little `:z` you put in there :)

Having run into the SELinux labels issue on Fedora myself, I'm aware of what it does. Might be worth putting a little code comment above here saying something to the effect of "We use `:z` on the host volume mount below to ensure the container runtime has the ability to read the contents contained in the host volume"

Reply:
indeed, added!